src/components/charts/largeScaleStructure/largeScaleStructureUtilities.js

```diff
@@ -44,10 +47,34 @@ export const getSeries = (name, options) => {
     symbolSize: 8,
     type: 'scatter',
     encode: {
-      x: 'RA',
-      y: 'Dec',
+      x: 'X',
+      y: 'Y',
```

Comment:
I was trying to make this file more generic, but seeing as this will probably be its own instance, I wouldn't mind just hard coding in the correct values. In this file's case, just removing the `encode` since it's being passed in the `options`